Apps/GatewayApi/test/unit/Services.Test/UserProfileServiceTests.cs

```diff
@@ -288,7 +288,7 @@ public async Task ShouldInsertUserProfile(DbStatusCode dbStatus, string registra
             // Act
             RequestResult<UserProfileModel> actualResult = await service.CreateUserProfile(
                     new CreateUserRequest
-                        { Profile = userProfile },
```

Comment:
This does not match our formatting template. I think this should be discussed with team... @MikeLyttle

Reply:
fixed